code-challenges.test.js

```diff
@@ -16,52 +16,163 @@
+
+//   ReferenceError: exampleFunction is not defined
 
 // b) Create the function that makes the test pass.
 
+// const exampleFunction = (arr) => {
+//     return arr.map((num) => {
+//         return num * 3
+//     })
+// }
+
+// PASS  ./code-challenges.test.js
+// exampleFunction
+//   ✓ takes in an array of numbers and returns an array with all the numbers multiplied by 3 (1 ms)
+
+const exampleFunction = (arr) => {
+    return arr.map(num => num * 3)
+}
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // Pseudo code:
+// input: object as an argument 
+// output: Even if numbers are even, or Odd if numbers are odd
+// pass object as an argument into the function
+// take value of number key 
+// return if the number is odd or even using an if else statement
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe("oddOrEven", () => {
+    it("takes an object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+        const object1 = { number: 15 }
+        // Expected output: "15 is divisible by three"
+        const object2 = { number: 0 }
+        // Expected output: "0 is divisible by three"
+        const object3 = { number: -7 }
+        // Expected output: "-7 is not divisible by three"
+        expect(oddOrEven(object1)).toEqual("Divisible by three")
+        expect(oddOrEven(object2)).toEqual("Divisible by three")
+        expect(oddOrEven(object3)).toEqual("Not divisible by three")
+    })
+})
+// FAIL  ./code-challenges.test.js
+//   exampleFunction
+    // ✕ takes a object as an argument and decides if the number inside it is evenly divisible by three or not. (1 ms)
+
+    // ● oddOrEven › takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+
+    // ReferenceError: oddOrEven is not defined
 
 // b) Create the function that makes the test pass.
 
+const oddOrEven = (object) => {
+    const num = object.number;
+    if (num % 3 === 0) {
+        return "Divisible by three";
+    } else {
+        return "Not divisible by three";
+    }
+}
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // Pseudo code:
+// input: Array of words
+// output: array of words with all letters capitalized 
+// function name: toCaps
+// use array of words as an argument for function toCaps
+// using a .map to itereate through each array and index
+// return an array with all words capitalized using .toUpperCase
```

Comment:
This pseudo code is incomplete. You explain how you are going to iterate, but there is no mention of `.charAt` or `.slice` and how they are working to do what you want them to.